### PR TITLE
[WebPortal] Generate a random initial password for new user

### PR DESCRIPTION
**Pull request by mslichao**
Generate a random initial password for new user

#3133

### CODE REVIEW

File: src/webportal/src/app/user/fabric/components/CustomPassword.jsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+import {TextField, Callout, DirectionalHint, DefaultButton, getTheme} from 'office-ui-fabric-react';
+
+function CustomPassword(props) {
+  const {onChange, onFocus, onBlur, componentRef, ...otherProps} = props;
+
+  useEffect(() => {
+    componentRef ? componentRef.current = passwordInputRef.current : null;
+    return () => {
+      if (componentRef) {
+        componentRef ? componentRef.current = null : null;
```
Comment (abuccts) [inline src/webportal/src/app/user/fabric/components/CustomPassword.jsx]: what's the ending `: null` for?

Comment (mslichao) [reply src/webportal/src/app/user/fabric/components/CustomPassword.jsx]: My fault, have fixed it.

File: src/webportal/src/app/user/fabric/components/CustomPassword.jsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    if (onChange) {
+      onChange(event, newValue);
+    }
+  };
+
+  const generateRandomPassword = () => {
+    return Array(8)
+      .fill('0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz')
+      .map((x) => x[Math.floor(Math.random() * x.length)])
+      .join('');
```
Comment (abuccts) [inline src/webportal/src/app/user/fabric/components/CustomPassword.jsx]: you can use `Math.random().toString(36).slice(-8)` to generate random string

Comment (mslichao) [reply src/webportal/src/app/user/fabric/components/CustomPassword.jsx]: Yes, `Math.random().toString(36).slice(-8)` is a great suggestion and more simple, but here I want the random password can have up-case/lower-case/number.

**Pull request merged**